### PR TITLE
[Azure Analysis Services] Onboarding SDK version 2017-07-14(https://github.com/Azure/azure-rest-api-specs/pull/1526)

### PR DESCRIPTION
**Pull request by taiwu**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by shahabhijeet** [comment]
@taiwu can you update this PR with swagger spec PR link.
Describe what this PR is all about?
There is no information about what this change is all about?

File: src/SDKs/AnalysisServices/AnalysisServices.Tests/AnalysisServices.Tests.csproj [hunk 1]
```diff
@@ -25,6 +25,9 @@
     <None Update="SessionRecords\AnalysisServices.Tests.ScenarioTests.ServerOperationsTests\CreateGetUpdateDeleteTest.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="SessionRecords\AnalysisServices.Tests.ScenarioTests.ServerOperationsTests\CreateServerWithGatewayTest.json">
```
Comment (shahabhijeet) [inline src/SDKs/AnalysisServices/AnalysisServices.Tests/AnalysisServices.Tests.csproj]: @taiwu please remove individual .json files entry and replace it with something like below
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Compute.Tests/Compute.Tests.csproj#L21

File: src/SDKs/AnalysisServices/AnalysisServices.Tests/Helpers/AnalysisServicesTestUtilities.cs [hunk 2]
```diff
@@ -39,6 +40,8 @@ public static class AnalysisServicesTestUtilities
         public static string DefaultResourceGroup = "TestRG";
         public static string DefaultServerName = "azsdktest";
         public static string DefaultLocation = "West US";
+        public static string DefaultGatewayResourceId = "/subscriptions/ba59a556-5034-4bbb-80b4-4c37cf1083e9/resourceGroups/TestRG/providers/Microsoft.Web/connectionGateways/azsdktest";
```
Comment (shahabhijeet) [inline src/SDKs/AnalysisServices/AnalysisServices.Tests/Helpers/AnalysisServicesTestUtilities.cs]: @taiwu please use test framework connection string to inject custom test data, do not hard code it.
https://github.com/Azure/azure-powershell/blob/preview/documentation/Using-Azure-TestFramework.md#5-change-test-environment-settings-at-run-time

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by taiwu** [comment]
ping

**Comment by shahabhijeet** [comment]
@taiwu please address my previous feedback about adding description about the PR.

**Comment by taiwu** [comment]
PR link https://github.com/Azure/azure-rest-api-specs/commit/bf407b7b72f209de3509ec1b3876d75df5fd459f
Another link for fixing issue https://github.com/Azure/azure-rest-api-specs/pull/1573

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**